app/src/main/java/com/semina/semi_na/view/login/LoginActivity.java

```diff
@@ -0,0 +1,106 @@
+package com.semina.semi_na.view.login;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.semina.semi_na.R;
+import com.semina.semi_na.data.remote.RetrofitClient;
+import com.semina.semi_na.data.remote.RetrofitInterface;
+import com.semina.semi_na.data.remote.request.LoginRequest;
+import com.semina.semi_na.data.remote.response.LoginResponse;
+import com.semina.semi_na.databinding.ActivityLoginBinding;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class LoginActivity extends AppCompatActivity {
+
+    private ActivityLoginBinding binding;
+    private EditText studentNumField;
+    private String studentNum;
+    private EditText passwordField;
+
+    private Button loginBtn;
+    private String password;
+
+    private RetrofitInterface retrofitInterface;
+
+    private SharedPreferences preferences;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        binding = ActivityLoginBinding.inflate(getLayoutInflater());
+        setContentView(binding.getRoot());
+
+        studentNumField = findViewById(R.id.login_studentNum_editText);
+        passwordField = binding.loginPasswordEditText;
+        loginBtn = binding.loginBtn;
+        retrofitInterface = RetrofitClient.getRetrofit().create(RetrofitInterface.class);
+        preferences = getSharedPreferences("UserInfo", MODE_PRIVATE);
+
+
+
+        loginBtn.setOnClickListener(view->{
+            studentNum = String.valueOf(studentNumField.getText());
+            password = String.valueOf(passwordField.getText());
+            Log.d("studentNum",studentNum);
+            Log.d("kathy",password);
```

Comment:
```suggestion
Log.d("LoginActivity", studentNum);
Log.d("LoginActivity", password);
```
로그 태그를 액티비티 이름으로 다는게 어떨까요
나중에는 태그 이름을 기억하기 힘들거 같아용

Reply:
좋은 아이디어입니당!